cv32e40s/tests/programs/custom/b_ext_test/test.yaml

```diff
@@ -4,8 +4,8 @@ description: >
     Simple sanity check for B extension instructions
 
 # Toolchain configurations
-riscv_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
-gnu_march:   rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
-corev_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
+riscv_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00_zicsr_zcmp
```

Comment:
remove all the *march except for cv_sw_march, they are deprecated

Reply:
agreed